### ISSUE TITLE
pandoc: "template" (line 642, column 111): unexpected "<" expecting letter

### ISSUE THREAD

**Issue by Tomas-Jankovic** [Wandmalfarbe/pandoc-latex-template#13]
Hi,

I have tried to use your template on Ubuntu 16.04.3 LTS and I have got this error message:
```
pandoc: "template" (line 642, column 111):
unexpected "<"
expecting letter
```
version of software:
* pandoc 1.16.0.2~dfsg-1
* texlive-latex-base 2015.20160320-1ubuntu0.1

**Comment by Wandmalfarbe** [comment]
Could you give me the document that you're trying to convert? I'll have a look then.

**Comment by Tomas-Jankovic** [comment]
command:
pandoc how_to_convert_markdown_to_pdf.md -o how_to_convert_markdown_to_pdf.pdf --from markdown -s --template eisvogel

[how_to_convert_markdown_to_pdf.md.gz](https://github.com/Wandmalfarbe/pandoc-latex-template/files/1422005/how_to_convert_markdown_to_pdf.md.gz)

**Comment by Wandmalfarbe** [comment]
Hm, that's strange. The error message tells me that pandoc can't parse something in the template (maybe a `$` because that character has a special meaning). When I run it on my mac everything is fine and I can't reproduce the problem.

Can it be that you downloaded the template somehow wrong, and it therefore contains invalid characters (possibly also something like a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark))?

You could try to [redownload the file from the GitHub generated zip here](https://github.com/Wandmalfarbe/pandoc-latex-template/archive/master.zip). It may also be possible, that the template does't work with the older pandoc version you are using. You could install the newer version like mentioned [on askubuntu](https://askubuntu.com/a/687676). This command should do the trick:

```
sudo wget https://github.com/jgm/pandoc/releases/download/1.19.2.1/pandoc-1.19.2.1-1-amd64.deb
sudo dpkg -i pandoc-1.19.2.1-1-amd64.deb
```

Unfortunately I don't have an Ubuntu installation ready at the moment.

**Comment by eben-roux** [comment]
I had the same issue and Wandmalfarbe is correct about the template being "corrupt".

Initially I didn't follow the instructions properly and added the template to my "source" folder where I had my markdown file.  Somehow pandoc used it and it was changed.

Once I actually copied it across to the proper user location I got the mentioned error since the template was altered into an unusable form.

I downloaded the template again and it is much smaller and the error was gone.

Just thought I'd mention that.
